ranger-tools/src/main/python/stress/stress-hbase-loadgenerator.py

```diff
@@ -0,0 +1,106 @@
+        with open(csv_outfile, "w") as f:
+            f.write("op,num_cf,num_keys,num_cols_per_cf,num_threads,time_taken,command,metadata,date_start,time_start,date_end,time_end\n")
+    assert type(num_threads_list) == list
+    assert type(num_rows_list) == list
+    for num_keys in num_rows_list:
+        for num_threads in num_threads_list:
+            if op_type == "multiput":
+                cmd = create_ltt_command_multiput(num_cols_per_cf=num_cols_per_cf,
+                                                  num_threads=num_threads,
+                                                  num_keys=num_keys,
+                                                  num_col_families=num_cf)
+            elif op_type == "multiget":
+                cmd = create_pe_command_multiget(multiget_batchsize=multiget_batchsize,
+                                                 num_threads=num_threads,
+                                                 num_keys=num_keys,
+                                                 num_col_families=num_cf)
+            else:
+                print("Invalid op_type")
+                exit(1)
+
+            datetime_start = datetime.now()
+            date_start_str = datetime_start.date()
+            time_start_str = str(datetime_start.time()).split(".")[0]
+            time_start = time.time()
+            ltt_out = subprocess.call(cmd, shell=True)
+            time_end = time.time()
+            datetime_end = datetime.now()
+            date_end_str = datetime_end.date()
+            time_end_str = str(datetime_end.time()).split(".")[0]
+            time_taken = time_end - time_start
+
+            print("cmd:", cmd)
+            print("LTT output:", ltt_out)
+            print("Time taken:", time_taken)
+            with open(csv_outfile, "a") as f:
+                if ltt_out != 0:
+                    time_taken = "non_zero_exit_code"
+                f.write(f'{op_type},{num_cf},{num_keys},{num_cols_per_cf},{num_threads},{time_taken},"{cmd}",{metadata},{date_start_str},{time_start_str},{date_end_str},{time_end_str}\n')
+                print(f"Written to file: {csv_outfile}")
+            # Sleep added so that the next command does not start immediately and any metric measurement such as heap useage can be captured more accurately
+            time.sleep(90)
+
+if __name__ == '__main__':
+    argparser = argparse.ArgumentParser("Generate LTT load and create report")
+    argparser.add_argument('-csv_output', '--csv_output', help='Full path to the csv output file', default="/root/ltt_output.csv", required=False)
+    argparser.add_argument('-metadata', '--metadata', help='Metadata to be added to the output file', default="no_cmd_line_metadata", required=False)
+    argparser.add_argument('-op_type', '--op_type', help='Type of operation to perform (multiget/multiput)', default="multiput", required=False)
+    args = argparser.parse_args()
+    increase_memory_for_loadgenerator()
+    login()
+    num_cf = 3
```

Comment:
Instead of hardcoding values, it is suggested to drive them from a config file. It makes the utility generic to use and more readable.